spec/models/collection_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Collection do
+  let(:user) { create(:user) }
+
+  describe 'add user to collection' do
+    it 'adds the collection owner to the collection' do
+      collection = described_class.create(title: collection_title_fixture, user:)
+      expect(collection.managers).to eq([user])
+    end
+
+    context 'when collection owner is already a manager' do
+      it 'does not add the collection owner to the collection' do
+        collection = described_class.create(title: collection_title_fixture, user:, managers: [user])
+        expect(collection.managers).to eq([user])
+      end
+    end
+
+    context 'when no collection owner' do
```

Comment:
What is the situation where there is a collection with no owner?

Reply:
Tests?

As currently implemented, a collection model doesn't have to have a user so wanted to err on the side of making sure it was covered.